src/features/student/selfKnowledge/components/interactions/formFields/CategoryElementDescriptionTextareaFormField/CategoryElementDescriptionTextareaFormField.test.ts

```diff
@@ -0,0 +1,135 @@
+import type { AddSelfKnowledgeCategoryElementForm } from '@/features/student/selfKnowledge/types/forms.types'
+import CategoryElementDescriptionTextareaFormField
+  from '@/features/student/selfKnowledge/components/interactions/formFields/CategoryElementDescriptionTextareaFormField/CategoryElementDescriptionTextareaFormField.vue'
+import { CategoryElementDescriptionTextareaStub } from '@/features/student/selfKnowledge/components/interactions/inputs/CategoryElementDescriptionTextarea/CategoryElementDescriptionTextarea.stub'
+import { BddTest } from '@avenirs-esr/avenirs-dsav/test-utils'
+import { useForm } from '@tanstack/vue-form'
+import { mount, type VueWrapper } from '@vue/test-utils'
+import { beforeEach, expect, vi } from 'vitest'
+
+const TestWrapper = {
+  components: {
+    CategoryElementDescriptionTextareaFormField
+  },
+  setup () {
+    const form = useForm({
+      defaultValues: { title: '', description: '', rating: null },
+      validators: {
+        onSubmit () {
+          return { fields: { description: undefined } }
+        }
+      }
+    }) as unknown as AddSelfKnowledgeCategoryElementForm
+
+    return { form }
+  },
+  template: `
+    <form @submit.prevent="form.handleSubmit">
+      <CategoryElementDescriptionTextareaFormField :form="form" />
+    </form>
+  `
+}
+
+BddTest().given('a self knowledge category element description textarea form field component', () => {
+  let wrapper: VueWrapper
+
+  const stubs = {
+    CategoryElementDescriptionTextarea: CategoryElementDescriptionTextareaStub
+  }
+
+  beforeEach(() => {
+    vi.clearAllMocks()
+    wrapper = mount(TestWrapper, {
+      global: { stubs }
+    })
+  })
+
+  BddTest().when('the component is mounted', () => {
+    BddTest().then('it should render the description textarea', () => {
+      const input = wrapper.findComponent({ name: 'CategoryElementDescriptionTextarea' })
+      expect(input.exists()).toBe(true)
+      const textarea = input.find('textarea')
+      expect(textarea.exists()).toBe(true)
+    })
+
+    BddTest().then('it should have the correct id', () => {
+      const input = wrapper.findComponent({ name: 'CategoryElementDescriptionTextarea' })
+      expect(input.props('id')).toBe('element-description')
+    })
+
+    BddTest().then('it should have empty initial value', () => {
+      const input = wrapper.findComponent({ name: 'CategoryElementDescriptionTextarea' })
+      expect(input.props('modelValue')).toBe('')
+    })
+  })
+
+  BddTest().when('the user types in the textarea', () => {
+    BddTest().then('it should update the form field value', async () => {
+      const input = wrapper.findComponent({ name: 'CategoryElementDescriptionTextarea' })
+      const textarea = input.find('textarea')
+
+      await textarea.setValue('My description text')
+      await wrapper.vm.$nextTick()
```

Comment:
Pareil pour la logique

```suggestion

    BddTest().and('the user types in the textarea', () => {
      beforeEach(() => {
        const input = wrapper.findComponent({ name: 'CategoryElementDescriptionTextarea' })
        const textarea = input.find('textarea')
        await textarea.setValue('My description text')
        await wrapper.vm.$nextTick()
      })
      
      BddTest().then('it should update the form field value', async () => {
```